### PR TITLE
pluralize log text

### PR DESCRIPTION
**Pull request by aruntk**
In the bootstrap log. `Bootstrapping 1 packages` -> `Bootstrapping 1 package`
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/BootstrapCommand.js [hunk 1]
```diff
@@ -129,18 +129,20 @@ class BootstrapCommand extends Command {
       if (err) {
         callback(err);
       } else {
-        this.logger.success("", `Bootstrapped ${this.filteredPackages.length} packages`);
+        this.logger.success("", `Bootstrapped ${this.filteredPackages.length} package${this.pluralize()}`);
```
Comment (evocateur) [inline src/commands/BootstrapCommand.js]: Let's just set this as an instance property near the beginning of the `initialize()` method, no need to have a one-liner instance method.
```
const filteredLength = this.filteredPackages.length;
this.packageCountLabel = `${filteredLength} package${filteredLength > 1 's' : ''}`;
```
in `execute()`'s inner callback:
```
this.logger.success("", `Bootstrapped ${this.packageCountLabel}`);
```
in `bootstrapPackages()`:
```
this.logger.info("", `Bootstrapping ${this.packageCountLabel}`);
```

**Comment by aruntk** [comment]
@evocateur Made the changes, please check. And sorry for the delay.

**Comment by aruntk** [comment]
@evocateur Any issues in the pr? It'd be helpful for contributors if you could comment the reason for closing a pr.

**Comment by evocateur** [comment]
Sorry, I finished a dramatic refactoring that moved the target of the PR. I did a bulk close of current PRs without checking, my apologies.

If you’d like to rebase, I’ll certainly merge it.

**Comment by aruntk** [comment]
@evocateur Resolved the conflicts. Please check.

File: commands/bootstrap/index.js [hunk 2]
```diff
@@ -36,6 +36,8 @@ class BootstrapCommand extends Command {
 
   initialize() {
     const { registry, rejectCycles, npmClient = "npm", npmClientArgs, mutex, hoist } = this.options;
+    const filteredLength = this.filteredPackages.length;
+    this.packageCountLabel = `${filteredLength} package${filteredLength > 1 's' : ''}`;
```
Comment (evocateur) [inline commands/bootstrap/index.js]: Er, you missed a `?` in the ternary.

Comment (aruntk) [reply commands/bootstrap/index.js]: @evocateur Sorry for that. Fixed in the latest commit.

File: commands/bootstrap/index.js [hunk 3]
```diff
@@ -104,7 +106,7 @@ class BootstrapCommand extends Command {
 
     // root install does not need progress bar
     this.enableProgressBar();
-    this.logger.info("", `Bootstrapping ${this.filteredPackages.length} packages`);
```
Comment (evocateur) [inline commands/bootstrap/index.js]: The `Bootstrapping` (begin) and `Bootstrapped` (end) labels have been reversed, needs to be restored.

Comment (evocateur) [reply commands/bootstrap/index.js]: And the snapshots need to be updated for the bootstrap integration test. `npm run integration -- -u`

Comment (aruntk) [reply commands/bootstrap/index.js]: @evocateur made the changes.

**Pull request merged**